javascript/javascript-6-object-literals.md

```diff
@@ -89,4 +90,10 @@ var loan = {
 - Work through [this interactive lesson on JavaScript Objects](https://www.codecademy.com/courses/introduction-to-javascript/lessons/objects/exercises/objects?action=resume_content_item).
 
 ### Check for Understanding
-Explain your completed code from the above Codecademy lesson to a partner.
+
+- Write down the syntax for declaring an Object.
+- What do you mean by Complex Objects. Give an example. 
```

Comment:
Shouldn't there be a question mark ?

Reply:
Do you mean ' What do you mean by Complex Objects? Give an example.'

Reply:
Yeah.

Reply:
Ohhk, Got it. I added a question mark